shell/server/demo-server.js

```diff
@@ -53,21 +53,15 @@ function cleanupExpiredUsers() {
   Meteor.users.find({ expires: { $lt: now } },
                     { fields: { _id: 1, loginIdentities: 1, lastActive: 1, appDemoId: 1 } })
               .forEach(function (user) {
+    console.log("delete demo user: " + user._id);
+    if (user.loginIdentities) {
+      globalDb.deleteAccount(user._id, globalBackend);
```

Comment:
This can trigger deletion of identities which maybe already be queued up to be deleted in a later step of this `forEach()` loop. That shouldn't cause any correctness problems, but it does seem a bit awkward.

Perhaps it would be better make two queries on `Meteor.users()`; in the first, iterate through and delete expired accounts, and in the second delete any remaining expired identities.